src/components/lib/Label.tsx

```diff
@@ -11,6 +11,7 @@ export interface IconProps {
      */
     fillStyle?: "solid" | "regular" | "light" | "duotone" | "brand";
     icon: string;
+    fa?: true | false;
```

Comment:
this should be `boolean`

Reply:
I fixed this in my fork, but I think the pull request needs to be reopened for the new commit to show up in the pull request.

Reply:
Please re-open the PR then